cji_smoke_test.sh

```diff
@@ -181,5 +181,19 @@ if [ "$MINIO_SUCCESS" = false ]; then
     exit 1
 fi
 
+JUNIT_TEST_OUTPUTS=(
+    "iqe-${CJI_NAME}-parallel.log"
+    "iqe-${CJI_NAME}-sequential.log"
+    "junit-${CJI_NAME}-parallel.xml"
+    "junit-${CJI_NAME}-sequential.xml"
```

Comment:
Do we always expect these 4 files to exist ? I know some plugins don't run parallel tests at all (that's the case for the majority).

Reply:
As discussed, these files should always exist.